### DISCUSSION TITLE
Hoppscotch

### DISCUSSION THREAD

**Discussion by zakafk** [coollabsio/coolify#3291] [Service Template Requests]
Hoppscotch should be in Services.

**Comment by zhorton34** [comment]
Commenting on this to back it as a great idea as an additional coolify service to add. Hoppscotch on coolify would be an awesome and very useful additional.

**Comment by shahzeb1** [comment]
Willing to work on this if someone starts a bounty.

**Comment by mredodos** [comment]
+1

**Comment by janbiasi** [comment]
✅ Hoppscotch is available as service

**Reply by alepouna** [reply]
@peaklabs-dev @Cinzya might be worth marking this as solved

**Comment by SpuriousGer** [comment]
I might be too dumb, but I am failing to install Hoppscotch with Email Auth, how do I get this done? The DB migration works but then it fails to actually start the backend.
